features/facebookImport/src/facebookImporter.jsx

```diff
@@ -90,32 +83,15 @@ const FacebookImporter = () => {
                     <Route exact path="/report">
                         <ReportView />
                     </Route>
-                    <Route exact path="/report/data-structure-info">
-                        <DataStructureInfoScreen />
-                    </Route>
-                    <Route exact path="/report/pictures-info">
-                        <PicturesInfoScreen />
-                    </Route>
-                    <Route exact path="/report/reaction-types-info">
-                        <PostReactionInfoScreen />
-                    </Route>
                     <Route exact path="/report/details">
                         <ReportDetails />
                     </Route>
-                    <Route exact path="/report/details/activities-info">
-                        <ActivitiesInfoScreen />
-                    </Route>
-                    <Route exact path="/report/details/messages-info">
-                        <MessagesInfoScreen />
-                    </Route>
-                    <Route exact path="/report/details/on-off-facebook-info">
-                        <OnOffFacebookInfoScreen />
-                    </Route>
-                    <Route exact path="/report/details/off-facebook-info">
-                        <OffFacebookInfoScreen />
-                    </Route>
                 </Switch>
             )}
+            {popUp &&
```

Comment:
This popUp is imported from ImporterContext, and it's substituting all those previous infoScreens?